.github/workflows/test.yml

```diff
@@ -31,6 +31,12 @@ env:
   CI_IMAGE_PR_TAR: image-pr.tar
   CI_DIST_IMAGES_OUTPUT: dist/images/_output/
 
+  # Select the OVN to build from
+  OVN_FROM: koji
+  # OVN_FROM: source
+  # OVN_REPO: https://github.com/ovn-org/ovn
+  # OVN_BRANCH:main
+  # OVN_COMMIT: HEAD
```

Comment:
Isn't these two a bit redundant? We can just have for example OVN_GITREF that can be an sha, tag or branch.

Reply:
Let's do that, I was adding some "smarteness" at kind.sh to calcalculate commit id from a branch so it always take the tip of the branch and force container rebuild if there are changes but I think we can also do that with this.

Reply:
I think container rebuild should be forced with it's own explicit paramerter, or depending on other parameters, for example `--deploy`

Reply:
> I think container rebuild should be forced with it's own explicit paramerter, or depending on other parameters, for example `--deploy`

What about always rebuilding when using the OVN_REPO/OVN_GITREF option at kind.sh ? 

> what about doing the trick in the makefile target

We are not using the makefile at kind.sh

Reply:
well I suggested before to use it

Reply:
> well I suggested before to use it

Do you mean adapting `make fedora` for these variables and call it from kind.sh and test.yaml ?

Reply:
@jcaamano looks like at "make fedora" we also generate the manifests with `./daemon.sh` should we remove that from there ? we are already doing it at kind.sh and test.yaml is also calling that.